Tests/test_parameter.py

```
@@ -1,29 +1,109 @@
-        assert isinstance(attr_id, int | str) 
+        assert isinstance(attr_id, int | str)
         assert attr_id in dpg.get_all_items()
 
-        value = param.get_value(attr_id)
-
-        assert value == 5
+        assert param.get_value(attr_id) == 5
 
         result = param.set_value(attr_id, 1)
-
+        assert result == True
```

Ruff (0.15.15)

[error] Avoid equality comparisons to `True`; use `result:` for truth checks

Replace with `result`

(E712)